concordium-consensus/src/Concordium/Kontrol/Bakers.hs

```diff
@@ -219,9 +221,10 @@ generateNextBakers ::
     ) =>
     -- | The payday epoch
     Epoch ->
+    Set.Set BakerId ->
     UpdatableBlockState m ->
     m (UpdatableBlockState m)
-generateNextBakers paydayEpoch bs0 = do
+generateNextBakers paydayEpoch suspendedBids bs0 = do
```

Comment:
This function is only called in one place, and there with `Set.empty`, so I don't think there's any need to parametrise it with the set.

Reply:
ok I removed the parameter and updated the docs, that this assumes no suspended validators.